### PR TITLE
Release Note for ONE 1.16.0

### PR DESCRIPTION
**Pull request by mhs4670go**
- Release Note for ONE 1.16.0

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
Please merge this PR after #6858 #6859 are landed.

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by lemmaa** [comment]
I just recognized that there was a problem with updating the release information in our documentation. I will solve this and separate PR.

File: docs/release/1.16/index.rst
Comment (mhs4670go) [reply docs/release/1.16/index.rst:6]: > I just recognized that there was a problem with updating the release information in our documentation.

@lemmaa Actually, as you said, all of other release note has the same contents about this. It's because those are not from automatic tool but from copy-paste. Could you let me know how to create `index.rst` by related tool?

Comment (lemmaa) [reply docs/release/1.16/index.rst]: @mhs4670go , unfortunately, it is not created automatically. You have to make it manually. :( 
I will try to make this series of processes more automated in the future when it comes to user document creation.

**Review by lemmaa — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**
